lib/tasks/import_datastream.rake

```diff
@@ -4,32 +4,24 @@
 # a class than a block and should be able to handle multiple tasks.
 # rubocop:disable Metrics/BlockLength
 namespace :ssg do
-  desc 'Update compliance DB with the latest release of the SCAP Security Guide'
-  task import_rhel: [:environment, 'ssg:sync_rhel'] do
-    # DATASTREAM_FILENAMES from openscap_parser's ssg:sync_rhel
-    begin
-      DATASTREAM_FILENAMES.flatten.each do |filename|
-        start = Time.zone.now
-        puts "Importing #{filename} at #{start}"
-        DatastreamImporter.new(filename).import!
-        puts "Finished importing #{filename} in #{Time.zone.now - start}"\
-             ' seconds.'
-      end
-    rescue StandardError => e
-      ExceptionNotifier.notify_exception(
-        e,
-        data: OpenshiftEnvironment.summary
-      )
-    end
+  desc 'Check if the latest SSG has been synced'
+  task check_synced: [:environment] do
```

Comment:
Do we have a coverage on this rake task?

Reply:
Now we do! Thanks for pointing out that oversight.